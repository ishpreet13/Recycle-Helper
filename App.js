import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import Routes from './app/navigation';

import AuthStack from './app/navigation/AuthStack';
import HomeScreen from './app/screens/HomeScreen';
import WelcomeScreen from './app/screens/WelcomeScreen';


export default function App() {
  return (
        //  <HomeScreen/>
      // <AuthStack/>  
      <Routes/>

    // <WelcomeScreen/>
    // <View style={styles.container}>
    //   <Text>Open up App.js to start working on your app!</Text>
    //   <StatusBar style="auto" />
    // </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
