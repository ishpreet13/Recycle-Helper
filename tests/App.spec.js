import React from "react";
import renderer, { act, create } from "react-test-renderer";

import App from "../App";

act(() => {
  describe("<App />", () => {
    let tree;
    it("has 1 child", async () => {
      act(() => {
        tree = create(<App />);
      });
      await expect(tree.toJSON().children.length).toBe(1);
    });
  });
});

act(() => {
  it("App should render correctly", async () => {
    let tree;
    act(() => {
      tree = create(<App />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
