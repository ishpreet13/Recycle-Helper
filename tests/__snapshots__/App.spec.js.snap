// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "rgb(242, 242, 242)",
            "flex": 1,
          },
          undefined,
        ]
      }
    >
      <RNSScreenContainer
        onLayout={[Function]}
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <RNSScreen
          activityState={2}
          forwardedRef={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            collapsable={false}
            style={
              Object {
                "opacity": 1,
              }
            }
          />
          <View
            accessibilityElementsHidden={false}
            closing={false}
            gestureVelocityImpact={0.3}
            importantForAccessibility="auto"
            onClose={[Function]}
            onGestureBegin={[Function]}
            onGestureCanceled={[Function]}
            onGestureEnd={[Function]}
            onOpen={[Function]}
            onTransition={[Function]}
            pointerEvents="box-none"
            style={
              Array [
                Object {
                  "display": "flex",
                  "overflow": undefined,
                },
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
              ]
            }
            transitionSpec={
              Object {
                "close": Object {
                  "animation": "spring",
                  "config": Object {
                    "damping": 500,
                    "mass": 3,
                    "overshootClamping": true,
                    "restDisplacementThreshold": 10,
                    "restSpeedThreshold": 10,
                    "stiffness": 1000,
                  },
                },
                "open": Object {
                  "animation": "spring",
                  "config": Object {
                    "damping": 500,
                    "mass": 3,
                    "overshootClamping": true,
                    "restDisplacementThreshold": 10,
                    "restSpeedThreshold": 10,
                    "stiffness": 1000,
                  },
                },
              }
            }
          >
            <View
              pointerEvents="box-none"
              style={
                Object {
                  "flex": 1,
                  "marginTop": 0,
                }
              }
            >
              <View
                collapsable={false}
                forwardedRef={[Function]}
                needsOffscreenAlphaCompositing={false}
                onGestureHandlerEvent={[Function]}
                onGestureHandlerStateChange={[Function]}
                style={
                  Object {
                    "flex": 1,
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateX": 0,
                      },
                    ],
                  }
                }
              >
                <View
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "overflow": "hidden",
                      },
                      Array [
                        Object {
                          "backgroundColor": "rgb(242, 242, 242)",
                        },
                        undefined,
                      ],
                    ]
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                        "flexDirection": "column-reverse",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "flex": 1,
                            "height": "100%",
                          }
                        }
                      >
                        <View
                          accessibilityIgnoresInvertColors={true}
                          style={
                            Object {
                              "alignItems": "center",
                              "flex": 1,
                              "justifyContent": "flex-end",
                            }
                          }
                        >
                          <Image
                            blurRadius={0.72}
                            source={1}
                            style={
                              Array [
                                Object {
                                  "bottom": 0,
                                  "left": 0,
                                  "position": "absolute",
                                  "right": 0,
                                  "top": 0,
                                },
                                Object {
                                  "height": undefined,
                                  "width": undefined,
                                },
                                undefined,
                              ]
                            }
                          />
                          <View
                            style={
                              Object {
                                "alignItems": "center",
                                "position": "absolute",
                                "top": 70,
                              }
                            }
                          >
                            <Image
                              source={1}
                              style={
                                Object {
                                  "height": 150,
                                  "width": 150,
                                }
                              }
                            />
                            <Text
                              style={
                                Object {
                                  "color": "#fff",
                                  "fontSize": 25,
                                  "fontWeight": "600",
                                  "paddingVertical": 20,
                                }
                              }
                            >
                              Recycle Helper
                            </Text>
                          </View>
                          <View
                            style={
                              Object {
                                "padding": 20,
                                "width": "100%",
                              }
                            }
                          >
                            <View
                              accessible={true}
                              focusable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "alignItems": "center",
                                  "backgroundColor": "lightblue",
                                  "borderColor": "#fff",
                                  "borderRadius": 25,
                                  "justifyContent": "center",
                                  "marginVertical": 10,
                                  "opacity": 1,
                                  "padding": 15,
                                  "width": "100%",
                                }
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#fff",
                                      "fontSize": 20,
                                      "fontWeight": "bold",
                                      "textAlign": "center",
                                      "textTransform": "uppercase",
                                    },
                                    Object {
                                      "fontSize": 20,
                                    },
                                  ]
                                }
                              >
                                Login
                              </Text>
                            </View>
                            <View
                              accessible={true}
                              focusable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "alignItems": "center",
                                  "backgroundColor": "lightblue",
                                  "borderColor": "#fff",
                                  "borderRadius": 25,
                                  "justifyContent": "center",
                                  "marginVertical": 10,
                                  "opacity": 1,
                                  "padding": 15,
                                  "width": "100%",
                                }
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#fff",
                                      "fontSize": 20,
                                      "fontWeight": "bold",
                                      "textAlign": "center",
                                      "textTransform": "uppercase",
                                    },
                                    Object {
                                      "fontSize": 20,
                                    },
                                  ]
                                }
                              >
                                Sign Up
                              </Text>
                            </View>
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNSScreen>
      </RNSScreenContainer>
      <View
        pointerEvents="box-none"
        style={
          Object {
            "height": 44,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "zIndex": 1,
          }
        }
      />
    </View>
  </RNCSafeAreaProvider>
</View>
`;
